lib/fdoprotctx.c

```diff
@@ -170,9 +170,8 @@ static bool fdo_prot_ctx_connect(fdo_prot_ctx_t *prot_ctx)
 		    prot_ctx->host_port, &prot_ctx->sock_hdl, prot_ctx->tls);
 		if (!ret) {
 			ret = connect_to_rendezvous(
-			    prot_ctx->host_ip, prot_ctx->host_dns,
-			    prot_ctx->host_port, &prot_ctx->sock_hdl,
-			    prot_ctx->tls);
+			    prot_ctx->host_ip, NULL, prot_ctx->host_port,
```

Comment:
what is the impact of removing prot_ctx->host_dns? and adding null?

Reply:
when providing invalid DNS (or NULL DNS) and valid IP it should connect with IP only so we make invalid DNS as NULL and go with IP